chapters/Chapter 1 — Untitled.md

```diff
@@ -18,18 +18,21 @@ Leanne turned just in time to see a crate of batteries break free of its binding
 
 Small batteries spilled from the crate as it smashed against the wall. Leanne did her best to ignore them as they pummeled her. Her legs and arms strained as she climbed into the cockpit.
```

Comment:
Does it feel faster-paced due to the single-clause per sentence, or does it feel slower?

I.E., would this feel faster-paced?

```
The crate smashed against the wall; small batteries spilled out. They pummeled her; Leanne did her best to ignore them: her legs ached and her arms strained as she pulled herself into the cockpit.
```

Too much?

```
The crate smashed against the wall and small batteries spilled out. They pummeled Leanne as she pulled herself into the cockpit, her arms shaking under the strain.
```

Not quite right?

```
The crate smashed against the wall. Small batteries spilled from it. Leanne couldn't feel them pummel her under the roar of her shaking arms and legs as she pulled herself into the cockpit.
```

Should those first two sentences be one?

```
The crate smashed against the wall; small batteries spilled from it. Leanne couldn't feel them pummel her under the roar of her shaking arms and legs as she pulled herself into the cockpit.
``` 

I feel like that's paced a bit more aggressively, but also a bit less humanly. It doesn't go as well with the second sentence. If I prefer that approach to narration, the second sentence should look more like this:

```
The crate smashed against the wall; small batteries spilled from it. Leanne couldn't feel them pummel her: the roar of her shaking arms and legs, straining to pull her into the cockpit, was too loud.
```

```
The crate smashed against the wall. Its small batteries pummeled her. She couldn't feel them under the roar of her arms and legs. They shook as they strained to pull her into the cockpit.
```

Or, mix up the order:

```
The crate smashed against the wall; small batteries spilled from it. Her arms and legs shook. They strained and roared to pull her into the cockpit.
```

I actually really like something about that version. But I also really don't like some other things.
They enveloped Leanne, pummeling her, their sides bruising her, their corners biting. 

```
The crate hit the wall. A swarm of energy cells spilled from it. Leanne ducked her head away. Tried to protect her eyes. She couldn't feel the cells as they enveloped her, their sides bruising, their corners biting. The roar from her arms and legs was too loud. They shook with strain. Pulled her up, slowly, into the cockpit.
```